apps/tailwind-components/assets/css/theme/molgenis.css

```diff
@@ -0,0 +1,127 @@
+:root[data-theme="molgenis"] {
+    --box-shadow-primary: 0px 10px 20px rgba(0, 0, 0, 0.1);
+    --box-shadow-search-input: none;
+    --box-shadow-pagination-gray: 0px 10px 20px rgba(0, 0, 0, 0.1);
+
+    --background-image-sidebar-gradient: linear-gradient(180deg, #0164C7 0%, rgba(1, 100, 199, 0) 86.02%);
+    --background-image-base-gradient: linear-gradient(180deg, #017FFD 0%, #0163C6 133.81%);
+
+    --opacity-background-gradient: 100;
+
+    --backgroud-color-button-primary: var(--color-yellow-500);
```

Comment:
In the InputRef PR, I've fixed this typo: `backgroud-*` in the main.css and tailwind config files.